scripts/cosm/deploy_erc20.js

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env node
```

Comment:
This is a super example script.
Not just for the CI, but to demo.

Great initiative!

Reply:
:)